recipes-extended/xen/xen-version.inc

```diff
@@ -1,3 +1,3 @@
 XEN_PV = "4.12.2-pre"
 XEN_BRANCH = "stable-4.12"
-XEN_SRCREV = "c28853456b51516514d36659494fbe7c1d983e2c"
+XEN_SRCREV = "${AUTOREV}"
```

Comment:
As AUTOREV moves forward, we can get out of sync with XEN_PV.  I don't know how we can handle that in an automated fashion, but XEN_PV could be set to just 4.12 or 4.12+.

Reply:
We can also fallback on the standard `_git+${SRCPV}` versioning of OE since we will now track a branch susceptible to change.

Reply:
@jandryuk , @eric-ch : ack, I'm looking into this.

Reply:
Branch refreshed with an update to the above logic to set `XEN_PV` with the `+git${SRCPV}` that is common in OE, to reflect that it is tracking the branch.

I've also removed the reference to `${XEN_PV}` where `FLASK_POLICY_FILE` was set in `xen-libxl.bb`, because that recipe does not package the flask policy file or require that variable. The flask policy filename from a policy built within the Xen tree has a filename that includes the full Xen version:
`xenpolicy-$(XEN_VERSION).$(XEN_SUBVERSION)$(XEN_EXTRAVERSION)`
-- which is affected by the switch to a tracking branch, since that version advances as the branch does. This can be addressed by changing the `FILES_${PN}-flask` definition to add globbing and it looks like the policy file handling in meta-virtualization's Xen recipe could be improved.